Sources/EpoxyLayoutGroups/Groups/Group.swift

```diff
@@ -98,10 +98,12 @@ extension Constrainable where Self: InternalGroup {
     return items[index]
   }
 
-  /// Shared implementation of `setItems(_ newItems:)`
-  func _setItems(_ newItems: [GroupItemModeling], animated: Bool) {
+  /// Shared implementation of `_setItems` is used across different groups
+  /// - Parameters:
+  ///   - newItems: the new set of items to set on the group
+  ///   - animation: enumeration to animate or not, with `LayoutGroupUpdateAnimation` available for animation
+  func _setItems(_ newItems: [GroupItemModeling], animated: Bool, animation: LayoutGroupUpdateAnimation?) {
```

Comment:
I believe we can make the `animation` parameter required right?

Reply:
Yes - and probably best to make it required so that there is an actual animation to render when animated is `true`!